src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaronContent.tsx

```diff
@@ -49,63 +48,66 @@ export const BuildBaronContent: React.FC<BuildBaronCoreProps> = ({
   return (
     <span data-cy="bb-content">
```

Comment:
This should be a `<div/>` https://www.tutorialspoint.com/What-is-the-difference-between-HTML-tags-div-and-span#:~:text=The%20element%20shows%20the,of%20text%2C%20images%2C%20etc.